ethereum-common/src/pb/sf.ethereum.substreams.v1.rs

```diff
@@ -0,0 +1,30 @@
+// @generated
```

Comment:
Do you need those new generated Protobuf definitions?

Reply:
I think you excluded the proto paths but forgot to remove the already generated files from git.